src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "../Footer/Footer.css";
+import {
+  CopyrightIcon,
+  GithubIcon,
+  LinkedinIcon,
+  TwitterIcon,
+} from "../../Assets/Svg/allsvg";
+
+const Footer = () => {
+  return (
+    <footer className="footer">
+      <div className="footer-header">
+        <h5 className="footer-title typography-h5">
+          Made with ❤️ by Zoya Tanweer
+        </h5>
+        <h6 className="footer-contact typography-h6">
+          Contact me
+          <ul>
+            <li className="list-item footer-item list-non-bullet">
+              <a href="https://github.com/zoyatanweer">
```

Comment:
Suggestion: Link tag to be used

Reply:
Using the Link tag will not take you to the external site and this is an external site.